packages/bot-engine/src/utils/html-helper.js

```diff
@@ -0,0 +1,11 @@
+export const importExternal = url => {
```

Comment:
We're using this in UI: `google-drive-store.js`, any reason why it's moved to `bot-engine`?

Reply:
@aaron-binary in case we need to have it in mobile, if we can not use it in mobile then we will move it here. there will be more moving when we share engine with mobile